website/tsconfig.json

```diff
@@ -18,7 +18,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "noImplicitAny": false
```

Comment:
hmmm, this isn't good, can you remove and solve the problems where you need implicit any's?

Reply:
I'll have a look

Reply:
Feel free to ping us on discord if you need advice on how to type stuff, it's a bit tricky at first.